tv.anilibria.anilibria.yml

```diff
@@ -0,0 +1,29 @@
+app-id: tv.anilibria.anilibria
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: /app/bin/AniLibria
+modules:
+  - name: anilibria
+    buildsystem: simple
+    build-commands:
+      - sed -i.bak 's/usr/${FLATPAK_DEST}/g' src/AniLibria.pro
+      - sed -i.bak 's/\/opt\/$${TARGET}\/bin/\/${FLATPAK_DEST}\/bin/g' src/AniLibria.pro
```

Comment:
doesn't qmake support using prefix? If you use `buildsystem: qmake` as suggested it should do the right thing instead of using obscure sed lines.